docs/installation/debian.md

```diff
@@ -49,17 +49,22 @@ from the new repository:
          $ apt-get purge lxc-docker*
          $ apt-get purge docker.io*
 
- 4. Add the new `gpg` key.
+ 4. Update package information; and ensure that APT can work with `https` method and that CA certificates are installed.
```

Comment:
Update package information, ensure that APT works with the `https` method, and that CA certificates are installed.